modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -76,7 +76,7 @@ The rootfs image is the same for FCP and DASD.
 ** Optional: To enable secure boot, add `coreos.inst.secure_ipl`
 
 ** For installations on DASD-type disks, complete the following tasks:
-... For `coreos.inst.install_dev=`, specify `/dev/dasda`.
+... For `coreos.inst.install_dev=`, specify `/dev/disk/by-id/scsi-<serial_number>`.
```

Comment:
for dasds this should be /dev/disk/by-path/ccw-<device id>, device id being e.g. 0.0.1000